pkg/cli/cmd/recipe/register/register.go

```diff
@@ -75,6 +75,7 @@ rad recipe register cosmosdb -e env_name -w workspace --template-kind bicep --te
 	_ = cmd.MarkFlagRequired("template-path")
 	cmd.Flags().String("resource-type", "", "specify the type of the portable resource this recipe can be consumed by")
 	_ = cmd.MarkFlagRequired("resource-type")
+	cmd.Flags().Bool("plain-http", false, "allow insecure connections to registry without SSL check")
```

Comment:
Same comments apply here that I left on the .tsp.

Reply:
updated it